src/cli/bootstrap/keys.ts

```diff
@@ -37,11 +37,17 @@ export async function generatePrivateKeyFromPassword(
 
 export const signer = EosioUtil.createSigner(defaultAntelopePrivateKey);
 
-export async function updateAccountKey(account: NameType, newPublicKey: PublicKeyType, addCodePermission = false) {
+export async function updateAccountKey(
+    account: NameType,
+    newPublicKey: PublicKeyType,
+    addCodePermission = false,
+    owner?: string
+) {
     const authority = Authority.fromKey(newPublicKey.toString());
 
     if (addCodePermission) authority.addCodePermission(account.toString());
 
+    console.log('owner', owner);
     await eosioContract.updateauth(account.toString(), 'active', 'owner', authority, signer);
-    await eosioContract.updateauth(account.toString(), 'owner', 'owner', authority, signer);
+    await eosioContract.updateauth(account.toString(), 'owner', owner ? owner : 'owner', authority, signer);
```

Comment:
when changing the owner key, the value of the owner variable is always 'owner'

this function should not change

Reply:
you want to make one account controlled by another? you will need to make a new function for this, similar to `updateAccountKey` but will use `Authority.fromAcccount()` instead to setup the account delegation.

Suggest you research how accounts on antelope work a bit more as well.